### DISCUSSION TITLE
Docker containers don't restart after reboot (except mail)

### DISCUSSION THREAD

**Discussion by tobika** [plausible/analytics#682] [Self-Hosted Support]
I see there is a `restart: always` defined for the mail container.
https://github.com/plausible/hosting/blob/0e9224017420417c3a45f89d1add4b1ccc2c29bd/docker-compose.yml#L5

Shouldn't this be defined in the plausible container to restart everything? I'm new to docker so maybe this doesn't work but makes sense to me. Thanks for the great project :)

**Comment by ukutaht** [comment]
Yes, on our production servers we actually use `restart: always`. Would make sense to add it to the hosting repo as well.

**Comment by winterrific** [comment]
Hi,

You can just add a restart policy to all necessary containers. 

Either:
_restart: unless-stopped_
Or:
_restart: always_

First does just restarting the container when it was running before the shutdown. Second restarts the container until it is destroyed.
